### PR TITLE
Add Hero Section

### PR DESCRIPTION
**Pull request by abregorivas**
Following changes were made
- Hero _include created
- Hero signup form created
- Set up hero as a collection
- Linked to index.html

### CODE REVIEW

File: _hero/hero.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: You can change LA for the better!
+message:
+  Can you design, write, or code? Are you an activist with an idea? You can help Los Angeles
+      live up to its potential at
+      Hack for LA. Everyone is welcome!
```
Comment (thekaveman) [inline _hero/hero.md]: Can you please clean up the spacing and line breaks?

File: _includes/hero.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<header class="hero">
+  <div class="hero-inner">
+    {% for item in site.hero %}
+    <h1>{{ item.title }}</h1>
+    <p class="type-featured">{{ item.message }}</p>
+    {% endfor %}
+    {% include forms/hero-signup.html %}
+    <p class="form-confirmation">
```
Comment (thekaveman) [inline _includes/hero.html]: Would this `form-confirmation` make more sense as part of the `forms/hero-signup` include? Presumably the two should always be used together, and some javascript is going to be acting on both.

File: _includes/hero.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<header class="hero">
+  <div class="hero-inner">
+    {% for item in site.hero %}
+    <h1>{{ item.title }}</h1>
+    <p class="type-featured">{{ item.message }}</p>
+    {% endfor %}
```
Comment (thekaveman) [inline _includes/hero.html]: If we're going to support multiple, we may want a wrapper around each for spacing etc.

**Review by thekaveman — CHANGES_REQUESTED** [review]

**Comment by abregorivas** [comment]
Re-based and pushed all changes.

**Review by thekaveman — APPROVED** [review]

**Pull request merged**